tests/ttnn/unit_tests/operations/test_unary.py

```diff
@@ -320,34 +325,32 @@ def test_logit(device, h, w, scalar):
 @pytest.mark.parametrize("h", [64])
 @pytest.mark.parametrize("w", [128])
 def test_pow(device, h, w, scalar):
-    run_unary_test_with_float(device, h, w, scalar, ttnn.pow, torch.pow, pcc=0.9)
+    run_unary_test_with_float(device, h, w, scalar, ttnn.pow, pcc=0.9)
 
 
-@pytest.mark.parametrize("scalar", [-0.5, -2.0])
 @pytest.mark.parametrize("h", [64])
 @pytest.mark.parametrize("w", [128])
 def test_relu_min(device, h, w, scalar):
-    run_unary_test_with_float(device, h, w, scalar, ttnn.relu_min, torch_relu_min)
```

Comment:
any reason why we are removing the scalar test param?

Reply:
need to pass upper_limit and lower_limit as kwargs to the golden function. so torch_relu_min and torch_relu_max can have their own test and separate golden function.